ansible/examples/deploy_grafana.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- hosts: all
+  user: vagrant
+  roles:
+    - nginx
+    - grafana
+  vars:
+     fqdn: "grafana.ceph.com"
+     app_name: "grafana"
+     development_server: true
+     # ssl hosts need to be defined so that all required ssl certs are put into
+     # place. This is specially important when one host responds for multiple
+     # other domains.
+     ssl_hosts:
+       - hostname: "grafana.ceph.com"
+         ssl_cert: "files/ssl/dev/ssl/ssl.crt"
+         ssl_key: "files/ssl/dev/ssl/ssl.key"
+       - hostname: "shaman.ceph.com"
+         ssl_cert: "files/ssl/dev/ssl/ssl.crt"
+         ssl_key: "files/ssl/dev/ssl/ssl.key"
+     # only needed when enabling Github Auth
+     # github_client_id: "111aaa222"
+     # github_client_secret: "qwerty1234"
+     #
+     # This has nothing to do with grafana, but uses nginx templating that is
+     # deployed on the same server. To prevent overwriting we set these here
+     # too. It also means that you must pass in both shaman node IPs
+     nginx_upstreams:
```

Comment:
If this needs to be the same in both ``deploy_grafana.yml`` and ``shaman_load_balancer.yml`` lets put it in a separate file and use ``vars_file`` to include it here in the playbook to avoid the duplication.